game/game.py

```diff
@@ -277,5 +276,10 @@ def is_game_over(self):
         return (False, GAME_OVER_NOT_MET)
 
     def get_current_player_name(self) -> str:
-        if self.remaining_moves > 0:
+        if self.remaining_moves > 0 and self.game_is_active and self.all_player_have_characters():
```

Comment:
This line could we a method "check_game_over()".  It will make more readable the code.
On the other hand, in the manager.py, it's calling a similar code in line 79.

Reply:
Yes, two functions make the same work but return different things. But it's true. It's material to refactor.
I create this ticket  to see this refactor in the future
[https://eventbrite.atlassian.net/browse/EDA-1673](https://eventbrite.atlassian.net/browse/EDA-1673)

Reply:
ok